src/rest_api/application.py

```diff
@@ -214,6 +233,39 @@ def post(self):
 
         return {'Response': 'The request has been sent and should be'
                             ' updated in Splunk as soon as it is done.'}
+
+
+@api.route('/analyze_similarity')
```

Comment:
I think this route also should be /similar

Reply:
Not sure i agree with this, but maybe

Reply:
Yep i agree with Anders